src/Microsoft.Management.Configuration.UnitTests/Tests/ConfigurationMixedElevationTests.cs

```diff
@@ -0,0 +1,176 @@
+// -----------------------------------------------------------------------------
+// <copyright file="ConfigurationMixedElevationTests.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.Management.Configuration.UnitTests.Tests
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Management.Configuration.UnitTests.Fixtures;
+    using Microsoft.Management.Configuration.UnitTests.Helpers;
+    using Microsoft.VisualBasic;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    /// <summary>
+    /// Unit tests for verifying the processor behavior for handling mixed elevation scenarios.
+    /// </summary>
+    [Collection("UnitTestCollection")]
+    [OutOfProc]
+    [OutOfProcOnly]
+    public class ConfigurationMixedElevationTests : ConfigurationProcessorTestBase
+    {
+        private readonly UnitTestFixture fixture;
+        private readonly ITestOutputHelper log;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ConfigurationMixedElevationTests"/> class.
+        /// </summary>
+        /// <param name="fixture">Unit test fixture.</param>
+        /// <param name="log">Log helper.</param>
+        public ConfigurationMixedElevationTests(UnitTestFixture fixture, ITestOutputHelper log)
+            : base(fixture, log)
+        {
+            this.fixture = fixture;
+            this.log = log;
+        }
+
+        /// <summary>
+        /// Verifies that creating a high integrity unit processor. Must run elevated.

```

Comment:
I don't want a test that will actually request admin.  The basic test should still use the test mode, with two units (one high, one medium) and a special resource that writes the PID into a target file so that you can ensure that they were routed to different set processors.